### DISCUSSION TITLE
Question about sentence transformers weights

### DISCUSSION THREAD

**Discussion by pablogranolabar** [ggml-org/llama.cpp#860] [General]
So is it possible to port the weights of other HuggingFace models such as sentence-transformers/all-MiniLM-L12-v2, to example generate embeddings for semantic textual similarity using llama.cpp? Ggml seems to indicate all of that work is happening here now, if that is any type of capability using the quantization code on other model architectures?

**Comment by jarcen** [comment]
Different architecture requires different code. It's possible but I don't know if we'll ever reach the point where this repo will support different architectures unrelated to llama.
